src/main/kotlin/com/chloe/kotlinserv/FullData.kt

```diff
@@ -0,0 +1,3 @@
+package com.chloe.kotlinserv
+
+data class FullData (val geoData: GeoData, val ipAddress: String?)
```

Comment:
it could be internal  private class of your Batch as you don't use it anywhere else